wdl/Scramble.wdl

```diff
@@ -12,32 +12,44 @@ workflow Scramble {
   input {
     File bam_or_cram_file
     File bam_or_cram_index
+    File original_bam_or_cram_file
+    File original_bam_or_cram_index
+    File counts_file
+    File manta_vcf
     String sample_name
     File reference_fasta
     File reference_index
     File regions_list
+
+    # Critical parameter for sensitivity/specificity
+    # Recommended values for aligners:
+    #   BWA-MEM: 90
+    #   DRAGEN-3.7.8: 60
+    Int alignment_score_cutoff
+
+    Float min_clipped_reads_fraction = 0.22
```

Comment:
What went into choosing this parameter? It seems pretty important. Is it worth considering an alternate approach  of setting it lower and then applying a filter to calls that don't meet this threshold, so that they'll be there in the VCF for cohorts where high sensitivity is desired?

Reply:
I agree this would be a useful feature. The only issue with increasing the threshold is cost. A lower threshold can bring in a lot more clusters which will drive up run time proportionally for all users. I think it would be a reasonable compromise to lower the threshold fed to Scramble by a tad (say to 0.15) so that the variants at least appear in the Scramble table, and then filter at 0.22 for the vcf. I don't think this is critical but I'm going to open a ticket for this.